sycl/include/sycl/kernel_bundle.hpp

```diff
@@ -8,6 +8,7 @@
 
 #pragma once
 
+#include <cstddef>                         // for std::byte
```

Comment:
Should be moved _after_ all SYCL includes, so that they don't accidentally rely on it being implicitly included